### PR TITLE
Switch between pc grounding search service and current aggregate search implementation by using an environment variable

### PR DESCRIPTION
**Pull request by metincansiper**

### CODE REVIEW

**Comment by metincansiper** [comment]
I am not able to test the reach side since in my all trials reach returned 0 entities (in the deployed unstable as well).

**Comment by maxkfranz** [comment]
Don't worry about the reach part.  We will no longer be using it in future to seed documents with entities and interactions anyway.

File: src/config.js [hunk 1]
```diff
@@ -37,6 +37,8 @@ let defaults = {
   MAX_SEARCH_SIZE: 50,
   BIOPAX_CONVERTER_URL: 'https://biopax.baderlab.org/convert/v2/',
   PC_URL: 'https://apps.pathwaycommons.org/',
+  GROUNDING_SEARCH_URL: 'http://localhost:3001',
```
Comment (maxkfranz) [inline src/config.js]: Maybe it should be marked as "base" url to be clearer?

**Comment by metincansiper** [comment]
Okay, I updated it as `GROUNDING_SEARCH_BASE_URL `

**Pull request merged**